config/overlays/test/qpext_image_patch.yaml

```diff
@@ -0,0 +1,2 @@
+data:
```

Comment:
if we want this to be default, we will need to add it to the kustomize.yaml as a patch. not sure we want that though. we probably want it to be configurable in some way 🤔

Reply:
We probably do not want this to be default at least for the initial version, also this Justinian layout is only patching kserve not knative. What you need to patch is actually the knative configmap.

Reply:
yeah my b - we would want something similar if we wanted default - since we dont though its fine as is. in the e2e test it applies the patch in the knative namespace